app/data_models/app_models.py

```diff
@@ -47,7 +47,7 @@ class DateTimeSchemaMixin:
     updated_at = fields.DateTime()
 
     @pre_dump
-    def set_date(self, data, many, **kwargs):
+    def set_date(self, data, **kwargs):
```

Comment:
Perfect example of our lint not working as expected 😞. Should be an `Unused argument`.